js/src/Ice/Ice.js

```diff
@@ -2,44 +2,40 @@
 // Copyright (c) ZeroC, Inc. All rights reserved.
 //
 
-const _ModuleRegistry = require("../Ice/ModuleRegistry").Ice._ModuleRegistry;
-
-module.exports.Ice = _ModuleRegistry.require(module,
-    [
-        "../Ice/Initialize",
-        "../Ice/Communicator",
-        "../Ice/CommunicatorI",
-        "../Ice/HashMap",
-        "../Ice/Object",
-        "../Ice/Value",
-        "../Ice/Long",
-        "../Ice/Logger",
-        "../Ice/LoggerI",
-        "../Ice/ObjectPrx",
-        "../Ice/BatchRequestQueue",
-        "../Ice/Properties",
-        "../Ice/PropertiesI",
-        "../Ice/IdentityUtil",
-        "../Ice/ProcessLogger",
-        "../Ice/Protocol",
-        "../Ice/Identity",
-        "../Ice/Exception",
-        "../Ice/LocalException",
-        "../Ice/BuiltinSequences",
-        "../Ice/StreamHelpers",
-        "../Ice/Promise",
-        "../Ice/EndpointTypes",
-        "../Ice/Locator",
-        "../Ice/Router",
-        "../Ice/Version",
-        "../Ice/Buffer",
-        "../Ice/ArrayUtil",
-        "../Ice/UnknownSlicedValue",
-        "../Ice/Process",
-        "../Ice/MapUtil",
-        "../Ice/ToStringMode"
-    ]).Ice;
+require("../Ice/ModuleRegistry");
```

Comment:
Is the order important here? If not it would be nice to sort alphabetically. Same question for other files.

Reply:
I sorted the requires, the order doesn't matter